Mathlib/Topology/Order/LowerUpperTopology.lean

```diff
@@ -281,6 +281,65 @@ instance (priority := 90) t0Space : T0Space α :=
 
 end PartialOrder
 
+section LinearOrder
+
+variable [LinearOrder α] [TopologicalSpace α] [IsLower α]
+
+lemma isTopologicalBasis_insert_univ_subbasis :
+    IsTopologicalBasis (insert univ {s : Set α | ∃ a, (Ici a)ᶜ = s}) :=
+  isTopologicalBasis_of_subbasis_of_inter (by rw [topology_eq α]; rfl)
+    (fun _ hs _ ht => by
+      simp at *
+      rcases hs with ⟨b, hb⟩
+      rcases ht with ⟨c, hc⟩
+      use b ⊓ c
+      rw [← hc, ← hb, Iio_inter_Iio])
```

Comment:
Can't you combine the `fun` and `rcases` into a single `rintro`?

Reply:
Yes, done. Thanks.